src/platform/backends/qemu/qemu_virtual_machine_factory.cpp

```diff
@@ -70,135 +71,189 @@ auto generate_tap_device_name(const std::string& vm_name)
     return hash_str;
 }
 
-bool run_network_cmd(QString cmd, QStringList args)
+auto generate_random_subnet()
 {
-    QProcess network_cmd;
-    network_cmd.setProgram(cmd);
-    network_cmd.setArguments(args);
+    std::stringstream subnet;
 
-    network_cmd.start();
-    network_cmd.waitForFinished();
+    unsigned seed = std::chrono::system_clock::now().time_since_epoch().count();
+    std::default_random_engine generator(seed);
+    std::uniform_int_distribution<int> distribution(0, 255);
+
+    while (true)
```

Comment:
Should this be finite instead? I'm wary of infinite loops, given the continuing conditions depend on external processes.
Maybe just try 100 possibilities or so?

Reply:
Sure, just following what LXD does when it generates a random subnet, but sure, let's bail out after 100 attempts:)